### PR TITLE
Css four column layout

### PR DESCRIPTION
**Pull request by cam-peck**
css-four-column-layout

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  <div class="container soft-gray-bg text-barlow">
+
+    <header class="row justify-center">
+      <div class="column-full">
+        <h2 class="text-extra-bold">How it Works</h2>
+      </div>
+    </header>
+
+    <main class="row justify-center text-center">
+      <div class="column-fifth">
```
Comment (Zhiwutian) [inline css-four-column-layout/index.html:24]: make sure you have an explicit width set for your column classes!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks great Cameron, just make sure to set an explicit width for your columns!

File: css-four-column-layout/index.html (hunk 1 above)
Comment (cam-peck) [reply css-four-column-layout/index.html:24]: Ahh - I totally forgot! I got lucky with the initial width of my container... Will fix now.

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**